Makefile

```diff
@@ -61,8 +62,13 @@ buf-lint: $(buf) examples/proto/api-common-protos
 	$(info [$@] linting protobuf schemas...)
 	@$(buf) lint
 
+protoc_plugins := \
+	$(protoc_gen_go) \
+	$(protoc_gen_go_grpc) \
+	$(protoc_gen_gapic_validator)
```

Comment:
What does this validate? Would it have caught invalid service?

Reply:
It caught the invalid resource types

Reply:
I'm thinking we can probably look at the rules in this validator, and port them over into our own linter - as part of moving the linter into here.